packages/features/ee/organizations/components/AddNewTeamsForm.tsx

```diff
@@ -110,7 +110,7 @@ export const AddNewTeamsForm = () => {
               onChange={(e) => handleInputChange(index, e)}
               addOnClassname="bg-transparent p-0 border-l-0"
               className={index > 0 ? "mb-2" : ""}
-              placeholder={t(`org_team_names_example_${index + 1}`) || t("org_team_names_example_1")}
+              placeholder={t(`org_team_names_example_${index + 1}`)}
```

Comment:
not sure what the || was trying to achieve so i removed it @Udit-takkar

Reply:
yeah not required